docs/guide/ru/README.md

```
@@ -114,12 +114,16 @@ $logger->setFlushInterval(100); // по умолчанию 1000
-Количество сообщений перед экспортом настраивается через `\Yiisoft\Log\Target::setExportInterval()`:
+Количество сообщений перед экспортом задаётся параметром конструктора `exportInterval`:
 
 ```php
-$target->setExportInterval(100); // по умолчанию 1000
+$target = new \Yiisoft\Log\StreamTarget(exportInterval: 100); // по умолчанию 1000
 ```
 
+> Сеттер `setExportInterval()` объявлен устаревшим с версии 2.2.2, используйте вместо него параметр конструктора.
+> То же касается `setCategories()`, `setExcept()`, `setLevels()`, `setFormat()`, `setPrefix()`,
+> `setTimestampFormat()` и `setEnabled()`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)